e2eAIOK/common/trainer/data/nlp/data_builder_squad.py

```diff
@@ -28,6 +26,7 @@ def prepare_dataset(self):
         dataset_train, train_examples, train_dataset, labels = build_dataset(is_train=True, args=self.cfg)
         dataset_val, val_examples, val_dataset, val_features, tokenizer = build_dataset(is_train=False, args=self.cfg)
         if 'teacher_model' in self.cfg and self.cfg.teacher_model != 'None':
+            from e2eAIOK.ModelAdapter.engine_core.distiller.utils import logits_wrap_dataset
```

Comment:
This also need a raise and print to instruct user install ModelAdaptor
I think in DeNas Train, dataloader is before init BertTrainer. Meaning you print in BertTrainer won't work since fail will happen here?
![image](https://user-images.githubusercontent.com/4355494/226381530-b8ea02d8-b32a-4ba5-9107-16222f722708.png)

Reply:
@tianyil1 , I see here is only checking if 'teacher_model' in self.cfg and self.cfg.teacher_model != 'None'. What I expected is if use env hasn't install modeladaptor, then this will also go failing without any suggestion?

Reply:
Good suggestions. I have refined it by placing the info into the dataloader once the MA modules first appear.